install/linux/usr/share/odemis/sim/sparc2-polarizer-sim.odm.yaml

```diff
@@ -0,0 +1,353 @@
+    properties: {
+        binning: [4, 16],
+    },
+}
+
+"Spectrograph": {
+    class: andorshrk.Shamrock,
+    role: "spectrograph",
+    power_supplier: "Power Control Unit",
+    init: {
+        device: "fake",
+        slits: {1: "slit-in", # INPUT_SIDE
+        },
+        bands: { # The filters in the filter-wheel
+            1: [320.e-9, 400.e-9], # pos -> m,m
+            2: [500.e-9, 550.e-9],
+            3: [400.e-9, 450.e-9],
+            6: "pass-through"
+        },
+        fstepsize: 10.9e-6, # m, size of one focus step
+    },
+    affects: ["Camera", "Spectrometer 1", "Spectral Camera", "Spectrometer 2"],
+}
+
+"Spectrograph focus": {
+    class: actuator.MultiplexActuator,
+    role: "focus",
+    children: {"z": "Spectrograph"},
+    init: {
+        axes_map: {"z": "focus"},
+    },
+    affects: ["Camera", "Spectrometer 1", "Spectral Camera", "Spectrometer 2"],
+}
+
+# Provide the mirror (aka port selector) of the spectrograph as a separate component
+"Spec CCD Flipper": {
+    class: actuator.FixedPositionsActuator,
+    role: "spec-det-selector",
+    children: {"rx": "Spectrograph"},
+    init: {
+        axis_name: "flip-out",
+        positions: {
+            0: ["Camera", "Spectrometer 1"],
+            1.5707963267948966: ["Spectral Camera", "Spectrometer 2"],
+        },
+    },
+    affects: ["Camera", "Spectral Camera", "Spectrometer 1", "Spectrometer 2"],
+}
+
+# Provide the filter wheel of the spectrograph as a separate component
+"Spec Filter Wheel": {
+    class: actuator.MultiplexActuator,
+    role: "filter",
+    children: {"band": "Spectrograph"},
+    init: {
+        axes_map: {"band": "band"},
+    },
+    affects: ["Camera", "Spectral Camera", "Spectrometer 1", "Spectrometer 2"],
+}
+
+"Optical Path Properties": {
+    class: static.OpticalLens,
+    role: "lens",
+    # Standard mirror config
+    init: {
+       mag: 0.31, # ratio
+       na: 0.2, # ratio, numerical aperture
+       ri: 1.0, # ratio, refractive index
+       pole_pos: [458, 519], # (px, px), position of the pole (aka the hole in the mirror)
+       x_max: 13.25e-3,  # m, the distance between the parabola origin and the cutoff position
+       hole_diam: 0.6e-3,  # m, diameter of the hole in the mirror
+       focus_dist: 0.5e-3,  # m, the vertical mirror cutoff, iow the min distance between the mirror and the sample
+       parabola_f: 2.5e-3,  # m, parabola_parameter=1/4f
+       rotation: -1.570796326795, # rad, 90° rotation between optical axis and SEM Y axis
+    },
+    affects: ["Camera", "Spectrometer 1", "Spectral Camera", "Spectrometer 2"]
+}
+
+# Controller for the motors moving the various parts of the optical box
+"Optical Actuators": {
+    class: tmcm.TMCLController,
+    role: null,
+    power_supplier: "Power Control Unit",
+    init: {
+        port: "/dev/fake6",
+        address: null,
+        axes: ["qwp", "linear", "l1", "l2", "slit"],
```

Comment:
You can make use of a new option in this class:
`unit: ["rad", "rad", "m", "m", "m"],`

And update the comment of `ustepsize`.